src/js/peteshow-controller.coffee

```diff
@@ -1,5 +1,6 @@
-_  = require('lodash')
-cs = require('calmsoul')
+_     = require('lodash')
+cs    = require('calmsoul')
+store = require('./peteshow-storage')
```

Comment:
I think an explicit requirement is nice, but I wouldn't add it until you actually need it . . . in case you don't ever need it :wink: